### ISSUE TITLE
System tray task

### ISSUE THREAD

**Issue by Jaffoo** [tryphotino/photino.NET#224]
How do I create a system tray task?

**Comment by Jaffoo** [comment]
> FYI - System Tray is not in scope, see the closed issue: #171

I find it, but how can I hide the window? The Photino does not support hidden windows.

**Comment by raddevus** [comment]
In your Program.cs when the main window is being created & instantiated you can try the following.

Set the size of the window to 0 x 0 with the SetSize(0,0) call.

```C#
// at the bottom of Program.cs right before the event loop is started (before WaitForClose() is called
        .Load("wwwroot/index.html"); // Can be used with relative path strings or "new URI()" instance to load a website.
window.SetSize(0,0);          
window.WaitForClose(); // Starts the application event loop
```

This is a workaround, but it will cause the window to be 0 pixels by 0 pixels and if the user clicks on the taskbar icon then no window will show.  Not perfect, but may get you to your solution.
